src/iosource/PktSrc.cc

```diff
@@ -385,6 +399,14 @@ void PktSrc::Process()
 			}
 		break;
 		}
+	default:
```

Comment:
Got it, thanks. 

Yeah, I'm expanding the cross-checks for non-IP a bit as I'm merging it.